stanford-earth/SE3_BLT
# EARTH-45: adding better_messages module.

# READY FOR REVIEW

# Summary
- Provides better styling & rendering for Drupal messages

# Needed By current sprint

# Steps to Test

1. Enable better_messages module
2. Go to `/admin/config/user-interface/better-messages` and configure to your liking
3. Do something that causes a message to appear on the matson theme, eg. webform submission

# Affected versions or modules
- Does this PR impact any particular module, version, or pull request?  *no*

# Associated Issues and/or People
- JIRA ticket https://stanfordits.atlassian.net/browse/EARTH-45
- Anyone who should be notified? @sherakama 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
